README.md

```diff
@@ -43,6 +43,25 @@ To generate images Adios uses Google Cloud's [Vertex AI image generation API](ht
 
 1. Now you can run or schedule the Adios services using the Adios menu
 
+## Using the Validation UI
+
+Adios provides an optional UI for users to check the generated images and approve/reject them before uploading them to the Google Ads account.
+
+You can use it as folows:
+
+1. Ensure that there is a set `Validated DIR` in the configuration Spreadsheet
```

Comment:
Just a proposition :)
"Please specify a value (e.g. VALIDATED) for the `Validated DIR` setting in the configuration sheet. This is the name for the directory in the GCS bucket, where approved images will be saved. Disapproved images will be saved to the directory specified in `Disapproved DIR`."

Reply:
Sounds great, thanks for the suggestion!